### PR TITLE
Add tsdb_k8s_queries challenge for serverless stress tests

### PR DESCRIPTION
**Pull request by inqueue**
Add an indexing and search workload for performing serverless stress testing. The track is designed to ramp up the indexing workload based on an increasing number of bulk clients for each `iteration_count`. The search workload remains constant throughout each iteration.

Highlights, focusing on `tsdb_k8s_queries/challenges/append-no-conflicts-rampup.json`:

* Add track parameters for the number of search and warmup iterations, target interval, and iteration count.
* Use queries from `append-no-conflicts`.
* Preload the target data streams with an initial parallel bulk indexing task.

- [ ] Needs docs

### CODE REVIEW

**Comment by inqueue** [comment]
@martijnvg, @gbanasiak this PR is ready for review. Can you give it a look?

**Review by martijnvg — APPROVED** [review]
lgtm

**Comment by inqueue** [comment]
Merged, docs to follow.

File: tsdb_k8s_queries/challenges/append-no-conflicts-rampup.json [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+    }
+    {# serverless-post-ingest-sleep-marker-start #}{%- if post_ingest_sleep|default(false) -%}
+    {
+      "name": "post-ingest-sleep",
+      "operation": {
+        "operation-type": "sleep",
+        "duration": {{ post_ingest_sleep_duration|default(30) }}
+      }
+    }
+    {%- endif -%}{# serverless-post-ingest-sleep-marker-end #}
```
Comment (gbanasiak) [inline tsdb_k8s_queries/challenges/append-no-conflicts-rampup.json:85]: Cosmetic remark: So far, the `post-ingest-sleep` operation was introduced in all the places where force-merge was present (as a simple replacement), in benchmarks where there's a clear-cut division between ingest and search. As this challenge is all about ramp-up and there's no clear moment when ingest stops I would skip this option.

**Pull request merged**